content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,44 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets
+---
+
+# Start by rotating access to various services and IAM accounts.
+
+- Access AWS
+- Rotate these specific IAM Users first
+    - `bosh_user`
+    - `concourse_user`
+    - `cf_user`
```

Comment:
yeah good call on both of these. I wonder if the runbook is the best place for the actual names? or if it's just better to say something like. **Look through `cg-provision`, and in AWS, and make sure any IAM users with Policies attached to them that can modify contents of buckets or the infrastructure need to be rotated first?